### PR TITLE
User can now logoff and log on

### LINKED ISSUE #48: Disable the application when not signed
- [ ] Stop a user from making a post or liking without signing in to the application

### PR DESCRIPTION
**Pull request by MichaelEWalker87**
### Description

* This stops a user from making a post or liking a post without being signed in 

### Related Issue

closes: #48,

### Motivation and Context

* We wanted to stop the user from making a post or liking a post without being signed in 

### How Has This Been Tested?

* The user should be able to make a user login with that user and like a post, the user should also be able to logout 

### Screenshots (if appropriate):

Screen Shot 2021-01-12 at 6 30 43 PM

Screen Shot 2021-01-12 at 6 31 44 PM

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,28 +17,33 @@ import YourPosts from './Your_Posts/Your_Posts';
 
 export default function App() {
   const [userData, setUserData] = useState({id: null, username: null});
-
-  render() {
+    console.log("APP Props", userData)
```
Comment (GreyMatteOr) [inline src/App.js:20]: REMOVE

**Review by Chriscastanuela — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -17,28 +17,33 @@ import YourPosts from './Your_Posts/Your_Posts';
...
       <section className='all-pages'>
-        <Header/>
+        <Header 
+          setUserData= {setUserData}
+          userData= {userData}
+        />
         <Route exact path='/' render={ () =>
           <section className="home">
             <Feed
               icon={dummyIcon}
```
Comment (GreyMatteOr) [inline src/App.js:30]: I don't know if we need this? I don't think we're displaying a userPic, are we?

File: src/Login/Login.js [hunk 3]
```diff
@@ -36,11 +35,12 @@ function Login(props) {
     )
   } else {
     console.log("data", data)
+    console.log("user data", data)
```
Comment (GreyMatteOr) [inline src/Login/Login.js:38]: REMOVE

**Review by GreyMatteOr — APPROVED** [review]
Do a little cleanup and APPROVED!

**Pull request merged**